code/datums/spells/touch_attacks.dm

```diff
@@ -21,7 +21,7 @@
 			if(!ChargeHand(target))
 				return 0
 	while(attached_hand) //hibernate untill the spell is actually used
-		charge_counter = 0
+		cooldown_handler.recharge_time++ // adds a tick onto the cooldown each tick
```

Comment:
This will make the cooldown longer the longer you use it, right? Should be reflected in the CL if that's the case

Reply:
Uhh, it shouldnt? It will mean that every tick, one tick of time is added to the cooldown. Which effectively resets the cooldown. Which is what it did before. 

While you had the hand out, it sets the charge counter to 0.